### ISSUE TITLE
docs.rs link 404s

### ISSUE THREAD

**Issue by untitaker** [assert-rs/libtest2#35]
Hi! [Very excited](https://unterwaditzer.net/2023/rust-test-parametrization.html) to see where this goes. Unfortunately I have no idea how to use this. Is it ready to use in any way?

Also, there's no getting started page or any example that indicates how basic tests would look like.

**Comment by epage** [comment]
Yes, because I've not yet published any packages.  The links point to where the documentation will be.

**Comment by epage** [comment]
Link is removed

**Comment by epage** [comment]
Well, I guess some links are still present

**Comment by epage** [comment]
Packages are now published
